src/pybind/mgr/dashboard/frontend/src/app/shared/components/info-panel/info-panel.component.less

```diff
@@ -0,0 +1,11 @@
+@import "../../../../../node_modules/bootstrap/less/variables.less";
```

Comment:
I don't like the idea of having mixed style formats. Could you change this file to `.scss`?